components/layout/Login.js

```diff
@@ -0,0 +1,108 @@
+import { useSession, signIn, signOut } from "next-auth/react";
+import Input from "../ui/Input";
+import Button from "../ui/Button";
+import styled from "styled-components";
+import { useRouter } from "next/router";
+import { useState } from "react";
+import { FaGithub, FaGoogle } from "react-icons/fa";
+import Link from "next/link";
+export default function Login() {
+  const { data: session } = useSession();
+  const router = useRouter();
+  const [credentials, setCredentials] = useState({
+    email: "",
+    password: "",
+  });
+
+  const handleSubmit = async (e) => {
```

Comment:
pleave don't use abbreviations like e. Also watch out for code consistency, so please use the function syntax for these functions.